docs/guides/operator-guide-azure-enclave.md

```diff
@@ -104,23 +98,22 @@ To deploy a new UID2 Private Operator for Azure, you'll need to complete the fol
 
 ### Download ZIP File and Extract Files
 
-The first step is to get set up with the deployment files you'll need.
+The first step is to get set up with the deployment files you'll need. Download the zip file linked in the table below for the Azure Download for the version you are installing. In most cases, you should install the latest version.
```

Comment:
The first step is to get set up with the deployment files you'll need. Download the ZIP file linked in the following table, Azure Download column, for the version you are installing. In most cases, you should install the latest version.

(ZIP is an acronym therefore all caps) (reworded to remove "below)

But: if we say "in most cases" we should say something about the other cases. otherwise simply say install the latest version. Reading that, I'd be concerned that mine might be a case where I should not install the latest version! How will they know?

Reply:
I changed it to this:
 The first step is to get set up with the deployment files you'll need. Download the ZIP file linked in the followi    ng table, Azure Download column, for the latest version.